cogs/eventstats.py

```diff
@@ -55,10 +55,16 @@ def __init__(self, bot):
                 )
 
         if not ctx.config:
-            # TODO Consider pulling most recent event and if time is between end of event and end of season, show stats.
-            return await ctx.send(
-                'It would appear that you aren\'t currently in an event. Try passing in an Event ID.'
-            )
+            query = "SELECT MAX(id) FROM events WHERE guild_id = $1 AND finish < NOW()"
```

Comment:
Since this is a group that happens the exact same for all 4 commands, would it be better to create a seperate function that does this once and just call that 4 times? It could return `True` if an event was found (and added to `ctx.config`), or return `False` if the error message was sent, and return from command if it's false.

Reply:
added get_recent_event function

Reply:
BTW, I wasn't sure how to add it to ctx.config without passing ctx to the function (which seemed cumbersome). So I just returned the event_id and did the event_config_id within the command. If there is an easier way to add to ctx.config within get_recent_event, I'm cool with that.